tools/cli/help.txt

```diff
@@ -1,19 +1,25 @@
 >>>
 (the first line of each entry is the summary for autogenerated command lists)
-Usage: meteor [--release <release>] [--help] <command> [args]
-       meteor help <command>
+Usage: meteor [--release <release>] [--help] <built-in command> [args]
+       meteor help <built-in command>
+       meteor <external command> [args]       
```

Comment:
What about `meteor <node | npm | npm command> [args]` so we make sure to put everything in the same pack and avoid using 3 lines of use?

I would add a small line to the explanation:
Meteor bundles the `node` and `npm` tools by default. You can use `meteor node` and `meteor npm` commands to ensure that the same version of `node` and `npm` which Meteor uses internally are being used.`